.github/workflows/release.yml

```diff
@@ -0,0 +1,113 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publishPreRelease:
+        description: "Publish a pre-release ?"
+        required: true
+        type: choice
+        options:
+          - "true"
+          - "false"
+        default: "true"
+      publishToMarketPlace:
+        description: "Publish to VS Code Marketplace ?"
+        required: true
+        type: choice
+        options:
+          - "true"
+          - "false"
+        default: "false"
+      publishToOVSX:
+        description: "Publish to OpenVSX Registry ?"
+        required: true
+        type: choice
+        options:
+          - "true"
+          - "false"
+        default: "false"
+jobs:
+  packaging-job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout vscode-knative
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
```

Comment:
According to https://github.com/redhat-developer/vscode-knative/pull/347 we can use version 20 now. If it works, I'd replace it with that here and in the release workflow below.

Reply:
^ Can this be Node 20 also ?